### ISSUE TITLE
ListPicker not working on API 29

### ISSUE THREAD

**Issue by vmutafov** [NativeScript/NativeScript#7785]
Hi,
It seems that the `ListPicker` is not working on API 29. The issue seems to be because of using a Java field through reflection which is 'hidden' in the API 29 SDK and causes a `NoSuchFieldException`: https://github.com/NativeScript/NativeScript/blob/4f39fb728b6bcacdafed38dd0a80137fcc9fb1e9/tns-core-modules/ui/list-picker/list-picker.android.ts#L67

**Comment by vmutafov** [comment]
Is this `selectorWheelPaintField` field actually used somewhere?

**Comment by manoldonev** [comment]
It is used by the property system https://github.com/NativeScript/NativeScript/blob/4f39fb728b6bcacdafed38dd0a80137fcc9fb1e9/tns-core-modules/ui/list-picker/list-picker.android.ts#L160 and https://github.com/NativeScript/NativeScript/blob/4f39fb728b6bcacdafed38dd0a80137fcc9fb1e9/tns-core-modules/ui/list-picker/list-picker.android.ts#L174

**Comment by manoldonev** [comment]
@vmutafov  I was able to reproduce the exception on api29 but only with tns-android@6.1.0. With tns-android@6.0.2 the same app works as expected on api29 — do you have any idea what causes the different behavior from runtime’s perspective?

**Comment by vmutafov** [comment]
Yeah, it's used, my bad I didn't see it. As for the question about the runtime version, 6.1.0 contains fixes in the runtime for using target and compile sdk 29. In the older runtimes, there was an issue when targeting sdk 29. I suppose you've tried it on an API 29 emulator but with an app built with target/compile sdk 29. Correct me if me presumption is wrong.

**Comment by pclokcer** [comment]
reduce app.gradle like :

`android {
  compileSdkVersion 29
  buildToolsVersion "29.0.2"
  defaultConfig {
    minSdkVersion 17
    generatedDensities = []
  }
  aaptOptions {
    additionalParameters "--no-version-vectors"
  }
}
`

and reduce to manifest like : 

android:versionCode="7"
android:versionName="505357"

and solved
